build/deploy.js

```diff
@@ -24,20 +24,30 @@ const deployToFirebase = (cb) => {
     return
   }
 
+  debug('Installing firebase-tools...')
+
+  exec('npm install -g firebase-tools', (error, stdout) => {
```

Comment:
If we are doing an async call to do this, we will need to do the two in succession, or we can't be sure that `firebase-tools` is installed before running the deploy script.

Is there a reason not to just keep the script right in the `travis.yml`